package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "book-exchange-book-service",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "prisma:migrate": "dotenv -e .env.development -- npx prisma migrate dev",
```

Comment:
This should be prisma:generate, then add --name.
Later, when we want to generate migrations, we can call:
yarn prisma:generate "Migration name"